api/views.py

```diff
@@ -206,8 +205,7 @@ class PuzzleTagViewSet(viewsets.ModelViewSet):
     serializer_class = PuzzleTagSerializer
 
     def get_queryset(self):
-        hunt_id = self.kwargs["hunt_id"]
-        return PuzzleTag.objects.filter(hunt__id=hunt_id)
+        return PuzzleTag.objects.all()
```

Comment:
Hmm this is a real behavior difference right? Do we imagine hunt-specific tags to be a thing? If not, we should remove the `hunt` field from PuzzleTags.

Reply:
There shouldn't be a behavior difference. This queryset is just the starting point that gets further filtered later (`destroy` filters it by `pk` in `self.get_object()` I think). With that said maybe I should make it puzzle.tags.all() to more accurately reflect what this class serves (tags for a specific puzzle).

As for `hunt` we definitely want tags across different hunts to be independent from each other so that's why this is here. It's technically duplicative since right now we assume that any tag is associated with at least one puzzle & you could get the hunt from that relationship, but eventually I was hoping to remove that constraint I think, for storing default tags.

Reply:
I see -- thanks!

Reply:
(I updated this to `PuzzleTag.objects.filter(puzzles__id=puzzle_id)` which is more precise)